botocove/cove_types.py

```diff
@@ -21,6 +22,7 @@ class CoveSessionInformation(TypedDict):
 
 
 CoveResults = List[CoveSessionInformation]
+FutureCoveResults = List["Future[CoveSessionInformation]"]
```

Comment:
probably not worth defining for one shot, but again - nitpick :)

Reply:
It was modelled after `CoveResults`, but I wasn't totally convinced either. I'll inline the type hint.

Reply:
It turns out CoveResults is the same one-shot type alias, so I added another commit to remove it as well.

I think "list of thing" is more readable than an alias to the same. The `CoveResults` alias had me thinking the type was something more complicated.

Reply:
Yeah, agreed - I think it was a bit more useful once upon a time :)